app/assets/javascripts/analytics_toolkit/google-analytics-universal-tracker.js

```diff
@@ -18,11 +18,6 @@
       sendToGa('set', 'anonymizeIp', true)
     }
 
-    function disableAdTracking () {
-      // https://support.google.com/analytics/answer/2444872?hl=en
-      sendToGa('set', 'displayFeaturesTask', null)
```

Comment:
According to the [docs](https://developers.google.com/analytics/devguides/collection/analyticsjs/tasks#disabling_a_task) linked to, setting to `null` disables this feature? This feels more explicit, rather than assuming that the default is to not have it enabled?

Reply:
Have realised that the `sendToGa(name + '.set', 'allowAdFeatures', false)` is all that's required to disable these features, and that the setting `displayFeaturesTask` to `null` is redundant. A rewrite of the commit message and PR description to include that explanation is all that's required. I'll approve in the meantime in expectation of that happening.